### PR TITLE
validation applied

### PR DESCRIPTION
**Pull request by Priyu714**
i have added validation and exeception in my code.

### CODE REVIEW

File: UserApplication/src/main/java/com/example/User/controller/UserController.java [hunk 1]
```diff
@@ -18,8 +19,7 @@ public class UserController {
     public UserService userService;
 
     @PostMapping("/creatuser")
-    public User createUser(@RequestBody User user ){
-        System.out.println("hello");
```
Comment (mikhi-mh) [inline UserApplication/src/main/java/com/example/User/controller/UserController.java:22]: remove these print these statements

File: UserApplication/src/main/java/com/example/User/exception/GlobalExceptionHandler.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@ControllerAdvice
+public class GlobalExceptionHandler extends ResponseEntityExceptionHandler {
+
+//    @Override
```
Comment (mikhi-mh) [inline UserApplication/src/main/java/com/example/User/exception/GlobalExceptionHandler.java]: You can remove this comment

File: UserApplication/src/main/java/com/example/User/service/UserService.java [hunk 3]
```diff
@@ -38,7 +38,6 @@ public String deleteUserById(Long id) {
     }
 
     public User updateUser(User user, Long id) {
-
         Optional<User> existingUser = userRepository.findById(id);
```
Comment (mikhi-mh) [inline UserApplication/src/main/java/com/example/User/service/UserService.java]: remove these unnecessary spacec

File: UserApplication/src/main/java/com/example/User/entity/User.java [hunk 4]
```diff
@@ -4,18 +4,35 @@
...
+
 
 @Document
 public class User {
 
     @MongoId
+    @NotNull(message = "Enter the valid Id ")
     private Long id;
 
+    @NotEmpty(message = "Should not null")
```
Comment (mikhi-mh) [inline UserApplication/src/main/java/com/example/User/entity/User.java]: Give proper message

File: UserApplication/src/main/java/com/example/User/exception/GlobalExceptionHandler.java (hunk 2 above)
Comment (Priyu714) [reply UserApplication/src/main/java/com/example/User/exception/GlobalExceptionHandler.java]: resolve

File: UserApplication/src/main/java/com/example/User/service/UserService.java (hunk 3 above)
Comment (Priyu714) [reply UserApplication/src/main/java/com/example/User/service/UserService.java]: removed

File: UserApplication/src/main/java/com/example/User/entity/User.java (hunk 4 above)
Comment (Priyu714) [reply UserApplication/src/main/java/com/example/User/entity/User.java]: Given Proper Message

**Comment by mikhi-mh** [comment]
you can merge this PR. I have approved.

**Pull request merged**